src/components/Calculator.js

```diff
@@ -1,18 +1,30 @@
 import React, { Component } from 'react';
 import Display from './Display';
 import Keyboard from './Keyboard';
+import calculate from '../logic/calculate';
+import DataObject from '../objects/DataObject';
 
 class Calculator extends Component {
   constructor(props) {
     super(props);
-    this.state = {};
+    this.state = {
+      dataObject: new DataObject(),
+    };
+  }
+
+  handleClick(key) {
+    this.setState((state) => ({
+      dataObject: { ...state.dataObject, ...calculate(state.dataObject, key) },
```

Comment:
- [ ] When an operator is clicked twice the app throws an error on the browser. Kindly incorporate error handling methods/conditions. 
![image](https://user-images.githubusercontent.com/64894927/129422725-8d4a138c-ba4d-4d79-bd5f-38aa0aa832e0.png)
- [ ] When a number is divided by zero, your app returns a result which is `0`. But it should return some message like `Infinity`. Kindly handle this behavior.